icepyx/core/granules.py

```diff
@@ -498,7 +498,7 @@ def download(self, verbose, path, restart=False):
         # (could create confusion depending on whether download was interrupted or kernel restarted)
         order_fn = ".order_restart"
         if os.path.exists(order_fn):
-            with open(order_fn, "r") as fid:
+            with open(order_fn) as fid:
```

Comment:
Why is being explicit not the preference here (even if "r" is default)?

Reply:
I think it's considered well-known or intuitive that read mode is the default? I don't think I fully agree with this. "explicit is better than implicit" seems like a good enough reason to disable this rule, but this is pretty low-stakes IMO, so I don't feel strongly.

The rule docs (https://docs.astral.sh/ruff/rules/redundant-open-modes/) say:

> ## Why is this bad?
> Redundant `open` mode parameters are unnecessary and should be removed to avoid confusion.

Whenever I see "it's unnecessary" used as a justification, it's a yellow-flag in my mind. We do lots of things that are unnecessary to make our work more accessible or provide other value.